docs/serving/creating-domain-mappings.md

```diff
@@ -5,23 +5,21 @@ weight: 64
 type: "docs"
```

Comment:
I just noticed this exist because I've searched the docs for such info and only found https://knative.dev/docs/serving/using-a-custom-domain-per-service/. Although, I think that the title fits both pages. Maybe they can be combined into one.

Reply:
I think maybe we should raise an additional issue for this since I don't want to expand the scope here too much; the initial reason for this PR was to make the private services docs more visible, so maybe could you raise one "to clean up custom domain docs"?

Reply:
Make sense. I'll create an issue for that.